<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormulaGen</class>
 <widget class="QMainWindow" name="FormulaGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formula</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="imageViewArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>360</width>
        <height>240</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="formulaPText">
        <property name="plainText">
         <string>(sin(x*2*pi)*sin(y*2*pi)+1+2*x+y)/5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generatePb">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelW">
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="widthSpin">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>64</number>
          </property>
          <property name="maximum">
           <number>4096</number>
          </property>
          <property name="value">
           <number>2048</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelH">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="heightSpin">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>64</number>
          </property>
          <property name="maximum">
           <number>4096</number>
          </property>
          <property name="value">
           <number>1536</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Image"/>
    <addaction name="actionPush_Image"/>
    <addaction name="actionPush_Image_as_Compose_Pattern"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_Image">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Image...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPush_Image">
   <property name="text">
    <string>Push Image as Depth Map</string>
   </property>
  </action>
  <action name="actionPush_Image_as_Compose_Pattern">
   <property name="text">
    <string>Push Image as Compose Pattern</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
